backend/main_original.py

```
@@ -0,0 +1,468 @@
+        log.info("s3 client initialized successfully")
+
+        athena_client = boto3.client(
+            'athena',
+            aws_access_key_id=aws_access_key_id,
+            aws_secret_access_key=aws_secret_access_key,
+            region_name=aws_region
+        )
+        log.info("athena client initialized successfully")
+    else:
+        log.error("Warning: AWS credentials not found")
+except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)